src/DataEditor.tsx

```diff
@@ -39,10 +37,10 @@ export default class DataEditor extends PureComponent<
     const { getValue, column, row, cell } = this.props;
     const value = getValue({ column, row, data: cell }) || "";
     return (
-      <div className="DataEditor">
+      <div className='DataEditor'>
```

Comment:
```suggestion
      <div className="DataEditor">
```
Don't switch the linting to single quotes unnecessarily